src/openfermion/measurements/_probability_distributions.py

```diff
@@ -0,0 +1,378 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""
+Classes to store probability distributions for estimation
+of random variables.
+"""
+import numpy
+from scipy import sparse
+from numpy import cos, sin, pi
+from openfermion.config import EQ_TOLERANCE
+
+
+class FourierProbabilityDist(object):
+    """
+    Stores an approximate probability distribution of our knowledge of
+    a quantum state in the eigenbasis of a unitary operator.
+
+    A generic quantum state may be written in the eigenbasis of a unitary
+    operator U,
+
+    $|\psi\rangle = sum_j a_j |phi_j\rangle$,
+
+    where psi is the quantum state, |phi_j\rangle are the eigenstates of U,
+    labeled by their eigenvalues ($U|phi_j\rangle = phi_j|phi_j\rangle$),
+    and a_j are the amplitudes.
+
+    This class stores a distribution P(phi_0, phi_1,... , A_0, A_1,...)
+    (where A_j = |a_j|^2) of our knowledge of these variables which may
+    then be updated by the results of experiments (in particular here,
+    QPE experiments).
```

Comment:
Since this class is meant to represent the distribution, don't mention anything about the possibility of updating it. If you do want to mention QPE, then say something like "the distribution can _determined_ by the results of experiments". The updating part should be handled solely by the estimator.

Reply:
I've removed the mention of updates from the docstring. The word update appears in a few other places in the code, but I feel where it does it's appropriate.